src/app/body/task-details/task-details.component.css

```diff
@@ -18,9 +18,41 @@
     background-color: #5A20F0;
 }
 
+#sprint {
+    color: #5A20F0;
+}
+
 @media (max-width: 991px) {
     .navbar-nav {
         float: right;
         flex-direction: row;
     }
+}
+
+.overlay {
+    position: fixed;
+    display: none;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgba(255, 255, 255, 1);
+    transition: opacity 500ms;
+    visibility: hidden;
+    z-index: 2;
+}
+
+.overlay:target {
+    visibility: visible;
+    opacity: 0.7;
+}
```

Comment:
Do we really need to force the UI for these tags

Reply:
These tags are required since the task details and logWork has been made as pop-up.